pkgs/by-name/me/meow/package.nix

```
@@ -0,0 +1,36 @@
+{
+  config,
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  ...
+}:
```

The additional arguments (`pkgs`, `config`, `...`) are not used and should be removed.

```suggestion
{
  lib,
  fetchFromGitHub,
  rustPlatform,
}:
```